regression-test/data/nereids_tpcds_shape_sf1000_p0/shape/query70.out

```diff
@@ -32,23 +32,22 @@ PhysicalResultSink
 --------------------------------------PhysicalProject
 ----------------------------------------filter((ranking <= 5))
 ------------------------------------------PhysicalWindow
---------------------------------------------PhysicalPartitionTopN
-----------------------------------------------PhysicalDistribute[DistributionSpecHash]
-------------------------------------------------PhysicalPartitionTopN
---------------------------------------------------PhysicalProject
-----------------------------------------------------hashAgg[GLOBAL]
-------------------------------------------------------PhysicalDistribute[DistributionSpecHash]
---------------------------------------------------------hashAgg[LOCAL]
-----------------------------------------------------------PhysicalProject
-------------------------------------------------------------hashJoin[INNER_JOIN] hashCondition=((store.s_store_sk = store_sales.ss_store_sk)) otherCondition=() build RFs:RF1 s_store_sk->[ss_store_sk]
---------------------------------------------------------------hashJoin[INNER_JOIN] hashCondition=((date_dim.d_date_sk = store_sales.ss_sold_date_sk)) otherCondition=() build RFs:RF0 d_date_sk->[ss_sold_date_sk]
-----------------------------------------------------------------PhysicalProject
-------------------------------------------------------------------PhysicalOlapScan[store_sales] apply RFs: RF0 RF1
-----------------------------------------------------------------PhysicalDistribute[DistributionSpecReplicated]
-------------------------------------------------------------------PhysicalProject
---------------------------------------------------------------------filter((date_dim.d_month_seq <= 1231) and (date_dim.d_month_seq >= 1220))
-----------------------------------------------------------------------PhysicalOlapScan[date_dim]
+--------------------------------------------PhysicalQuickSort[LOCAL_SORT]
```

Comment:
Why the origin shape missing PhysicalQuickSort?

Reply:
origin shape use two phase PhysicalPartitionTopN with a data shuffle between them. This pr correct the slot id issue, so the new shape use one phase PhysicalPartitionTopN + PhysicalQuickSort instead, and no need to shuffle data.